must-gather/Dockerfile

```diff
@@ -13,21 +13,32 @@ RUN curl --location --output velero.tgz https://github.com/openshift/velero/arch
     curl --location --output restic.tgz https://github.com/openshift/restic/archive/refs/heads/${RESTIC_BRANCH}.tar.gz && \
     tar -xzvf restic.tgz && cd restic-${RESTIC_BRANCH} && \
     CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -mod=mod -ldflags '-extldflags "-static"' -o /restic github.com/restic/restic/cmd/restic && \
-    cd .. && rm -rf restic.tgz restic-${RESTIC_BRANCH}
+    cd .. && rm -rf restic.tgz restic-${RESTIC_BRANCH} && \
+    curl --location --output oc.tgz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.16/openshift-client-linux-${TARGETARCH}-rhel9.tar.gz && \
+    tar -xzvf oc.tgz && rm -rf oc.tgz README.md kubectl
 
 FROM registry.access.redhat.com/ubi9/go-toolset:latest AS gobuilder
 
 RUN go install -v github.com/google/pprof@latest
 
-FROM quay.io/openshift/origin-must-gather:latest AS builder
+# FROM registry.access.redhat.com/openshift4/ose-cli-rhel9:v4.16 AS builder
+
+# RUN which oc
+# # /usr/bin/oc
+# RUN oc version
+# # Client Version: v4.2.0-alpha.0-2197-g91ba77a
+# # Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
```

Comment:
will remove comments prior to merge, but want to check if it ok to bump `oc` version. It was 4.2, now will be 4.16

Reply:
@mateusoliveira43 , I see no reason not to _**especially**_ as old as you've identified it still is.

FWIW, 
```
$ podman run -it --rm --entrypoint /usr/bin/oc registry.redhat.io/oadp/oadp-mustgather-rhel9:1.4.0 version
Client Version: 4.16.0-202405281506.p0.g12a2e98.assembly.stream.el9-12a2e98
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
```

Reply:
thanks for the info, will update then!